src/sections/layer_and_mask_information_section/groups.rs

```diff
@@ -19,12 +19,11 @@ impl Groups {
     /// Add a group to the list of groups, making it last in the order.
     pub fn push(&mut self, group: PsdGroup) {
         self.group_ids_in_order.push(group.id);
-
         self.groups.insert(group.id, group);
     }
 
     /// Get the group ID's in order (from bottom to top in a PSD file).
-    pub fn group_ids_in_order(&self) -> &Vec<u32> {
+    pub fn group_ids_in_order(&self) -> &[u32] {
```

Comment:
Can revert. If we want to introduce a breaking change it should be justified.

Reply:
see comment above